README.md

```diff
@@ -0,0 +1,13 @@
+# pac-cmd
+
+A command line tool to change proxy auto-config settings of operation system. Binaries included. Simply `make` to build it.
+
+# Usage
+
+```sh
+pac [on  <pac url> | off]
+```
+
+*  **Mac**: Have to `sudo` as setting pac is an privileged action.
+
+*  **Windows**: Install [MinGW-W64](http://sourceforge.net/projects/mingw-w64) to build pac, as it has up to date SDK headers we require.
```

Comment:
Also, should we consider switching back to MinGW in systray?  I found Visual Studio useful to create the DLL, but I guess one can actually create DLLs with MinGW [too](http://www.mingw.org/wiki/sampledll).